### ISSUE TITLE
Pitop Rasp4 and RaspOS not working

### ISSUE THREAD

**Issue by pikowski** [rricharz/pi-top-setup#10]
Hallo,
i have problem running pt-device-manager on my PITOP, now running with a Raspberry Pi 4 and latest Raspberry Pi OS.
Former Raspbian is running well on the same hardware once i am starting from my sd-card with Raspbian. Command pt-devices gives back: Unknown host device. No change of brightness possible and no battery status there. Is there anything what i can do?

**Comment by rricharz** [comment]
I don‘t think so. Pi-top does not appear to have fixed their device manager for the Pi 4. I have unfortunately no pi-top which can be used to test pi-top‘s software with a Pi 4.

**Comment by m-roberts** [comment]
Hi - for now, please use the package in our repository. See our knowledge base article here: https://knowledgebase.pi-top.com/knowledge/pi-top-and-raspberry-pi-os

We are undergoing some significant architectural changes, so these instructions are subject to change, but you should always be able to find them at this link.

Thanks!
